src/graphorm/index.ts

```diff
@@ -266,6 +266,9 @@ const obj = new GraphORM({
       },
     },
   },
+  SourceCategory: {
+    requiredColumns: ['createdAt'],
```

Comment:
Why is this one required only?

Reply:
The `createdAt` is needed for the pagination, technically you can have everything missing and not break the query.